src/include/daos_api.h

```diff
@@ -298,6 +298,27 @@ daos_pool_set_attr(daos_handle_t poh, int n, char const *const names[],
 		   void const *const values[], size_t const sizes[],
 		   daos_event_t *ev);
 
+/**
+ * List a pool's containers.
+ *
+ * \param[in]	poh	Pool connection handle.
+ * \param[in,out]
+ *		ncont	[in] \a conts length in items.
```

Comment:
"\a conts" needs to be updated for the actual parameter names?

Reply:
Fixed.